src/utils/env.utils.ts

```diff
@@ -11,3 +11,4 @@ export const IOS_APP_ID = getEnv('IOS_APP_ID');
 export const ANDROID_APP_ID = getEnv('ANDROID_APP_ID');
 
 export const TEMPLE_WALLET_API = getEnv('TEMPLE_WALLET_API');
+export const SEGMENT_ANALYTICS = getEnv('SEGMENT_ANALYTICS');
```

Comment:
rename to `SEGMENT_ANALYTICS_KEY`

Reply:
and add it to .github/workflows/testflight-build.yml `Create .env file` step. (otherwise production build will not include it)